### PR TITLE
Added a slack notifier

### PR DESCRIPTION
**Pull request by richardtape**
with configurable options for the message attachment and settings. Also updated the readme to reflect this additional notifier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ Behatch notifiers provide notifiers for behat.
 
     $ composer require "behatch/notifiers"
 
+For Slack notifier, additionally
+
+    $ composer require "maknz/slack"
```
Comment (sanpii) [inline README.md:15]: You can add this package in composer.json as suggested requirement.

File: src/Slack.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Behatch\Notifier;
+
+class Slack extends Notifier
+{
+    static private $url;
+	static private $settings;
```
Comment (sanpii) [inline src/Slack.php]: Tabulation, please use 4 spaces instead.

File: src/Slack.php [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+			return;
+		}
+
+		// If there's a prefix, prepend it to the title of the message
+        if (self::$prefix !== null) {
+            $title = '[' . self::$prefix . '] ' . $title;
+        }
+
+		// Fire up the Slack Client with the given webhook url and settings
+		$client = new \Maknz\Slack\Client(self::$url, self::$settings);
```
Comment (sanpii) [inline src/Slack.php]: To prevent an horrible backtrace, check in the constructor if this class exist and print a comprehensible message for the user.

**Comment by richardtape** [comment]
Thanks for the notes. I've adjusted as per your recommendations.

**Comment by sanpii** [comment]
Merged, thank you :smiley:

**Pull request merged**